### PR TITLE
All components and spaces visible

### PR DESCRIPTION
**Pull request by ykulbak**

### CODE REVIEW

**Comment by ykulbak** [comment]
This test case should be tested in Bimsync and Solibri before its accepted.

**Review by cschladetsch — COMMENTED** [review]
Punctuation at end of paragraphs and bullet points is inconsistent.

Indented bullets are not indented.

File: Test Cases/v3.0/Visualization/all components and spaces visible/unzipped/bcf.version [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<Version VersionId="3.0">
+  <DetailedVersion>3.0</DetailedVersion>
```
Comment (pasi-paasiala) [inline Test Cases/v3.0/Visualization/all components and spaces visible/unzipped/bcf.version:3]: Remove detailed version.

**Review by pasi-paasiala — CHANGES_REQUESTED** [review]
-1

Detailed version must go. Imports fine otherwise.

**Comment by ykulbak** [comment]
@pasi-paasiala @jasollien ready for review. I've applied all comments

**Review by pasi-paasiala — APPROVED** [review]
+1

**Pull request merged**